packages/graphql/src/translate/authorization/where/create-authorization-jwt-payload-predicate.ts

```diff
@@ -64,7 +62,7 @@ export function createJwtPayloadWherePredicate({
     return Cypher.and(...predicates);
 }
 
-function createNestedPredicate({
+function createNestedPredicateLegacy({
```

Comment:
Additionally, we could mark this as `@deprecated`

```suggestion
/**
 * @deprecated
 **/
function createNestedPredicateLegacy({
```